tests/_test_msui/test_msui.py

```diff
@@ -315,22 +314,21 @@ def test_plugin_export(self, save_file):
             assert os.path.exists(save_file[0])
             os.remove(save_file[0])
 
-    @pytest.mark.skip("needs to be refactored to become independent")
     @mock.patch("PyQt5.QtWidgets.QMessageBox")
     @mock.patch("mslib.msui.msui_mainwindow.config_loader", return_value=export_plugins)
     def test_add_plugins(self, mockopen, mockbox):
```

Comment:
The default is not to have further import / export plugins. 
The name is misleading or the result should show that plugins can become added

Reply:
I don't understand what you mean. The test adds import and export plugins and then checks that they (or at least "one more") shows up in the respective lists. It also removes them again and checks that they no longer show up then, so I guess it could be renamed to `test_add_and_remove_plugins` or split into two tests.

Reply:
seems I readed it not completly. Thanks for asking.